tests/models/multimodal/generation/test_common.py

```diff
@@ -851,6 +851,12 @@
                 limit_mm_per_prompt={"image": 4},
             )
         ],
+        marks=[
+            # TODO: remove skip when upgrading Transformers to v4.57.2
+            pytest.mark.skip(
+                "Broken in Transformers v4.57.1, should be fixed in v4.57.2"
+            )
```

Comment:
```suggestion
            pytest.mark.skipif(
                Version(TRANSFORMERS_VERSION) < Version("4.57.2")
            )
```
I think we can skip the test based on transformers version.